features/providers/means_report.feature

```diff
@@ -21,6 +21,7 @@ Feature: Means report
       | h2  | Property, savings and other assets |
       | h3  | Property |
       | h3  | Vehicles |
+      # | h2  | Which bank accounts does your client have? | * TODO: awaiting flow change for journey (AP-3536)
```

Comment:
I think we should be creating tasks in JIRA rather than writing TODOs.

Reply:
fixed after rebase, i was waiting for